playbooks/wordpress-stack-install.yml

```diff
@@ -1,16 +1,23 @@
 ---
-- name: Install MariaDB, nginx, PHP-FPM, and WordPress
+- name: install MariaDB, nginx, PHP-FPM, and WordPress
   hosts: all
   become: yes
 
-  vars:
-    wp_db_name: wordpress-foss-magic
-    wp_db_password: 'What is a more secure way to manage passwords in a public playbook?'
-    wp_db_user: wordpress-admin
-
   roles:
     - ../roles/common-centos/7
     - ../roles/mariadb
     - ../roles/nginx
     - ../roles/php-fpm
-    - ../roles/wordpress
+    - role: ../roles/wordpress
+      # Disable All Updates
+      # By default automatic updates are enabled, set this value to true to
+      # disable all automatic updates
+      auto_up_disable: false
+      # Define Core Update Level
+      #   true  = Development, minor, and major updates are all enabled
+      #   false = Development, minor, and major updates are all disabled
+      #   minor = Minor updates are enabled, major updates are disabled
+      core_update_level: true
+      wp_db_name: wordpress-foss-magic
+      wp_db_password: 'What is a secure way to manage passwords in public infrastructure?'
```

Comment:
You could use an inline vault string (https://docs.ansible.com/ansible/2.6/user_guide/vault.html#use-encrypt-string-to-create-encrypted-variables-to-embed-in-yaml) with either a vault password provided at runtime or a vault password file (https://docs.ansible.com/ansible/2.6/user_guide/vault.html#providing-vault-passwords)